resources/ec2/v1alpha1/igw.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
this filename should be `resources/ec2/v1alpha1/internetgateway.yaml` to match the resource name